<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1384</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover {
	border:2px solid #1573a0;
	border-radius:5px;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>0</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string>ПОСТЕР</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:none;
font: 16pt &quot;Segoe UI&quot;;
}</string>
   </property>
   <property name="title">
    <string>Удалить фильм</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QLineEdit" name="removeName_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_16">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Фильм ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>531</width>
     <height>541</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	border:none;
	margin-top:5px;
	font: 16pt &quot;Segoe UI&quot;;
}
QLabel{
font: 11pt &quot;Segoe UI&quot;;
}
</string>
   </property>
   <property name="title">
    <string>Добавить фильм</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pick_movie">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>85</y>
      <width>23</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:1px;
background-color:#45494c;
}
QPushButton :hover {
border:1px solid blue;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pick_poster">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>50</y>
      <width>274</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>273</width>
      <height>0</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Выбрать постер</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_8">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>47</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Постер </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_6">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>85</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильм</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>122</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>122</y>
      <width>271</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>86</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="genre_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>158</y>
      <width>272</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>268</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>158</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Жанр</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_4">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>194</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Режиссер</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="director_label">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>194</y>
      <width>271</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="duration_label">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>230</y>
      <width>271</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>минуты</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_5">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Длительность</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>269</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Год выпуска</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="year_box">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>267</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Segoe UI&quot;;
font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1900</number>
    </property>
    <property name="maximum">
     <number>2024</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
    </property>
    <property name="value">
     <number>1943</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_movieName_3">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>267</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Рейтинг</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rating_box">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>267</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="url_movie_label">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>85</y>
      <width>249</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Выберите фильм</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="description_textEdit">
   <property name="geometry">
    <rect>
     <x>763</x>
     <y>350</y>
     <width>611</width>
     <height>131</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addmovie_button">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>490</y>
     <width>331</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>631</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>740</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="poster_label">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>221</width>
     <height>311</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>283</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid aqua;</string>
   </property>
   <property name="text">
    <string>ПОСТЕР</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>1266</x>
     <y>585</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
